.github/ISSUE_TEMPLATE/Standard-Investigation-Issue-Template.md

```diff
@@ -19,21 +20,30 @@ labels: Investigation
 
 or
 
-- [ ] A solution has been discovered but it was deemed not worth it
+- [ ] A solution has been discovered but it was deemed not worth it, and reasoning is documented on this issue
 
 ## Subject Domain Experts
 < list of subject domain experts for this domain. If the subject expert from the end-user perspective is not the author, an end-user expert should be listed here. The Design, Dev, and QA teams should add experts here as appropriate. >
 
 ## Progress
-### Reviewed Issues - completed by SM & Team Leads
-- [ ] Inspiration and Value complete
-- [ ] Initial Acceptance Criteria Defined
-- [ ] The person who will Approve story identified
+### Fully Created Issues - completed by author
+- [ ] Inspiration and Value completed
+- [ ] Acceptance Criteria Specified
+- [ ] Approver Identified
+- [ ] Labels attached
+- [ ] SME's provided
+
+### SME Reviewed Issues - completed by SM & Team Leads
+- [ ] Description approved
+- [ ] Acceptance Criteria Finalized
+- [ ] Approver correct
+- [ ] Appropriate labels attached
+- [ ] SME list complete
```

Comment:
This requirement seems to be missing from the other templates, even though all the templates seem to have that section. Also, I feel we should make sure this acronym (SME) and the section match up. Currently, the section is called Subject Domain Experts, not Subject Matter Experts. I'm fine with either, as long as they match.

Reply:
gotcha. Will add the SME checkbox everywhere and will change the section name to say Subject Matter Expert